### DISCUSSION TITLE
Meeting week of 27th Jun

### DISCUSSION THREAD

**Discussion by jonfreedman** [finos/technical-oversight-committee#6] [Polls]
Testing out discussions, what is the preferred day for our next meeting?

**Comment by ColinEberhardt** [comment]
Thanks for starting the poll @jonfreedman - one thing we should consider is whether we take a poll each time, or whether people would prefer a fixed day / time.

We could set up another poll to discuss! Or perhaps easier to just add to the agenda for our next meeting #11

**Reply by jonfreedman** [reply]
Am not wedded to polls in any way, was just testing out the discussions features 😊

**Comment by eminty69** [comment]
Just as an FYI, Friday 1st is a public holiday in Canada

**Comment by ColinEberhardt** [comment]
This meeting has now taken place, minutes here https://github.com/finos/technical-steering-committee/pull/14
